src/test/kotlin/org/opensearch/replication/integ/rest/UpdateAutoFollowPatternIT.kt

```diff
@@ -81,6 +84,8 @@ class UpdateAutoFollowPatternIT: MultiClusterRestTestCase() {
             }, 60, TimeUnit.SECONDS)
         } finally {
             followerClient.deleteAutoFollowPattern(connectionAlias, indexPatternName)
+            followerClient.waitForReplicationStart(leaderIndexName, waitForShardTask)
+            followerClient.waitForReplicationStart(leaderIndexNameNew, waitForShardTask)
```

Comment:
These are probably better added in the try block. That way we also clearly ensure replication was started as well.

Reply:
We can probably keep it after `leaderIndexNameNew` is created on the leader side.

Reply:
done.